src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -100,4 +102,48 @@ install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ./perf_
   WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin/)")
 install(CODE "message(\"-- Created symlink: perf_client -> ./perf_analyzer\")")
 
+# Configure doctest
+include(ExternalProject)
+find_package(Git REQUIRED)
+
+ExternalProject_Add(
+    doctest
+    PREFIX ${CMAKE_BINARY_DIR}/doctest
+    GIT_REPOSITORY https://github.com/doctest/doctest.git
+    TIMEOUT 10
+    UPDATE_COMMAND ${GIT_EXECUTABLE} pull
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+    INSTALL_COMMAND ""
+    LOG_DOWNLOAD ON
+)
+
+# Expose required variable (DOCTEST_INCLUDE_DIR) to parent scope
+ExternalProject_Get_Property(doctest source_dir)
+set(DOCTEST_INCLUDE_DIR ${source_dir}/doctest CACHE INTERNAL "Path to include folder for doctest")
+
+target_include_directories(perf_analyzer PUBLIC ${DOCTEST_INCLUDE_DIR})
+target_compile_definitions(perf_analyzer PUBLIC DOCTEST_CONFIG_DISABLE)
```

Comment:
disable unit tests for perf analyzer binary. none of the unit testing code is compiled for the perf analyzer binary. it's only compiled for the unit test runner binary